scripts/start_civ.sh

```diff
@@ -66,6 +66,8 @@ GUEST_STATIC_OPTION="\
  -serial chardev:charserial0 \
  -device virtio-blk-pci,drive=disk1,bootindex=1 \
  -device intel-iommu,device-iotlb=on,caching-mode=on \
+ -fsdev local,security_model=none,id=fsdev0,path=./share_folder \
```

Comment:
The share folder is already enabled in: https://github.com/projectceladon/vm_manager/blob/c11556329adf60330a1f5a590acf14d49c66b993/scripts/start_civ.sh#L229 why need this? and this might have conflict? Please check.